lms/djangoapps/philu_overrides/helpers.py

```diff
@@ -71,18 +71,30 @@ def get_course_next_classes(request, course):
     from student.models import CourseEnrollment
     from opaque_keys.edx.locations import SlashSeparatedCourseKey
     from course_action_state.models import CourseRerunState
+    from openedx.features.course_card.helpers import get_course_open_date
+
+    courses = []
```

Comment:
can we rename this to something like future_courses?

Reply:
This method just returns course open date and not any courses

Reply:
@emraan004 how about to define a method to get course_runs as we defined `get_course_current_class` to get current classes.